### PR TITLE
😎 Create a Button to add a Task

### LINKED ISSUE #40: US1-3:  MIT Button
Create a flashcard repository and enable the addition of more cards. 

Add + button to add more tasks.

### PR DESCRIPTION
**Pull request by 3dcantaloupe**
# Abstract

Please include a summary of the changes and the related issue. At minimum, describe:

- What problem does this PR solve?
- How does this PR solve the problem?
- Be specific and avoid vague language like "make it better".
- What kinds of changes did you make to solve the problem?
- What kinds of new classes did you add?
- What are their responsibilities?
- How do they integrate or interoperate?

This PR essentially adds the required items for the specific user story task 1-3: adding the button that shows the add MIT interface. I added xml files for the action bar, and modified the mainactivity to incorporate this change. I also added the icon file to the res assets. The action bar adds a bar on top for the app, and a place for the add button to be. Also added CreateMitFragment.java which is the boilerplate for the add interface.

Basically, explain to your teammates reviewing this PR how to navigate the code you're submitting. 

# Issues

Mention any issues that are being closed by this PR, using the `Closes #<issue number>` syntax so
that they are linked to this PR and automatically closed when the PR is merged.

Closes #40 

## Type of Change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)

# Testing

Adding tests in 1-6, as there are not many tests to be done on the button with no functionality yet

# Checklist

Perform a self-review using the following checklist. Some items may not apply to your PR if it 
is for a bug fix. You can check an item by replacing the [ ] with [x].

- [x] **The code is understandable.**
  - [x] The code is follows formatting conventions (spacing, indents, etc).
  - [x] The code makes use of names that sound like behavior.
  - [x] The code is commented in hard-to-understand areas.
  - [x] The code contains changes to documentation if needed.

- [x] **The code is well structured.**
  - [x] The code is organized into logical components.
  - [x] The components sound like the behavior (requirements) they implement.
  - [x] The components have clearly defined single responsibilities.

- [x] **The code is correct.**
  - [x] The changes include tests that prove the fix is effective or that the feature fully works.
  - [x] The changes generate no new failed tests, build errors, or warnings.

# Pending Items

Is there anything else that needs to be done before this PR can be merged?

### CODE REVIEW

**Review by pink10000 — COMMENTED** [review]
why is there so much commented out code?

File: app/src/main/java/edu/ucsd/cse110/successorator/ui/mitList/dialog/CreateMitFragment.java
Comment (Tankstar03) [inline app/src/main/java/edu/ucsd/cse110/successorator/ui/mitList/dialog/CreateMitFragment.java:1]: Delete this file

**Review by Tankstar03 — CHANGES_REQUESTED** [review]

**Comment by 3dcantaloupe** [comment]
> why is there so much commented out code?

Before we split this US task into two since this is too difficult as one task. I stopped working on that, and commented it out, as this US task only requires the icon. But the code is still useful for US1-6, so I just kept it commented in for now.

**Comment by pink10000** [comment]
> > why is there so much commented out code?
> 
> Before we split this US task into two since this is too difficult as one task. I stopped working on that, and commented it out, as this US task only requires the icon. But the code is still useful for US1-6, so I just kept it commented in for now.

hmm maybe we should delete it then since it's not being used

**Comment by 3dcantaloupe** [comment]
> > > why is there so much commented out code?
> > 
> > 
> > Before we split this US task into two since this is too difficult as one task. I stopped working on that, and commented it out, as this US task only requires the icon. But the code is still useful for US1-6, so I just kept it commented in for now.
> 
> hmm maybe we should delete it then since it's not being used

Yes, that is the case, however we do need it in US 1-6, so I feel like we should still keep it in there.

**Comment by pink10000** [comment]
> > > > why is there so much commented out code?
> > > 
> > > 
> > > Before we split this US task into two since this is too difficult as one task. I stopped working on that, and commented it out, as this US task only requires the icon. But the code is still useful for US1-6, so I just kept it commented in for now.
> > 
> > 
> > hmm maybe we should delete it then since it's not being used
> 
> Yes, that is the case, however we do need it in US 1-6, so I feel like we should still keep it in there.

ok good foresight, nice

**Review by pink10000 — APPROVED** [review]

**Review by Tankstar03 — APPROVED** [review]

**Pull request merged**